app/components/page_nav_component.html.erb

```diff
@@ -0,0 +1,40 @@
+
+<style>
```

Comment:
Can we move this to a separate scss file. I only inlined some of the CSS in the prototype to minimise changes.

Also wondering whether "analysis-page-nav" might be better scoping for the class name?

Reply:
Yes - that's on my todo list at the top :-)

I thought the nav could be reusable for other sections of the site - hence the broader scoping?

Reply:
Moved the css out for now but hope to move it later into a component css file.